src/dataframerow/utils.jl

```diff
@@ -135,14 +129,47 @@ function row_group_slots(cols::Tuple{Vararg{AbstractVector}},
             groups[i] = gix
         end
     end
-    return ngroups, rhashes, gslots
+    return ngroups, rhashes, gslots, false, true
+end
+
+function row_group_slots(cols::Tuple{CategoricalVector},
+                         hash::Val{false},
+                         groups::Union{Vector{Int}, Nothing} = nothing,
+                         skipmissing::Bool = false)::Tuple{Int, Vector{UInt}, Vector{Int}, Bool, Bool}
+    col = cols[1]
+    @assert groups === nothing || length(groups) == length(col)
+
+    # If missings are to be skipped, they will all go to group 1
```

Comment:
and this group will be skipped; otherwise missings to group `ngroups`. Right?

Reply:
I've expanded the comment a bit as that's confusing.